policygenerator/policy-sets/community/openshift-plus-setup/opp-settings.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v1
+data:
+  vmwareReplicas: "6"
+  vmwareCPU: "8"
+  vmwareMemory: "32768" 
+  awsReplicas: "2"
+  awsInstanceType: m6a.2xlarge
+  us-east-1-4.10: ami-0c72f473496a7b1c2
+  us-east-1-4.11: ami-0722eb0819717090f
+  us-east-1-4.12: ami-0fe05b1aa8dacfa90
+  us-east-1-4.13: ami-0624891c612b5eaa0
+  us-east-1-4.14: ami-0b56cb92505dea7ed
+  us-east-1-4.15: ami-0b56cb92505dea7ed
+  us-east-1-4.16: ami-057df4d0cb8cbae0d
+  us-east-2-4.10: ami-09e637fc5885c13cc
+  us-east-2-4.11: ami-026e5701f495c94a2
+  us-east-2-4.12: ami-0ff64f495c7e977cf
+  us-east-2-4.13: ami-0dc6c4d1bd5161f13
+  us-east-2-4.14: ami-0dc6c4d1bd5161f13
+  us-east-2-4.15: ami-0b577c67f5371f6d1
+  us-east-2-4.16: ami-0f736c64d5751d7d3
+  zone1: a
+  zone2: b
+  zone3: c
+kind: ConfigMap
+metadata:
+  name: opp-settings
+  namespace: policies
```

nit: Moved the metadata to the top. I find the alphabetical field ordering unconventional.
```suggestion
kind: ConfigMap
metadata:
  name: opp-settings
  namespace: policies
data:
  vmwareReplicas: "6"
  vmwareCPU: "8"
  vmwareMemory: "32768" 
  awsReplicas: "2"
  awsInstanceType: m6a.2xlarge
  us-east-1-4.10: ami-0c72f473496a7b1c2
  us-east-1-4.11: ami-0722eb0819717090f
  us-east-1-4.12: ami-0fe05b1aa8dacfa90
  us-east-1-4.13: ami-0624891c612b5eaa0
  us-east-1-4.14: ami-0b56cb92505dea7ed
  us-east-1-4.15: ami-0b56cb92505dea7ed
  us-east-1-4.16: ami-057df4d0cb8cbae0d
  us-east-2-4.10: ami-09e637fc5885c13cc
  us-east-2-4.11: ami-026e5701f495c94a2
  us-east-2-4.12: ami-0ff64f495c7e977cf
  us-east-2-4.13: ami-0dc6c4d1bd5161f13
  us-east-2-4.14: ami-0dc6c4d1bd5161f13
  us-east-2-4.15: ami-0b577c67f5371f6d1
  us-east-2-4.16: ami-0f736c64d5751d7d3
  zone1: a
  zone2: b
  zone3: c
```

---

ok, i fixed that too.  Thanks!